js/map.js

```diff
@@ -645,5 +652,33 @@ function validateLayerName(map, layer) {
     return true;
 }
 
+function computeLaneProbabilities(point, possibleLanes) {
+	let lanes = point.clusters;
+	let laneProb = new Map();
+
+	// Probability scaling
+	let totalProb = 0;
+
+	// Compute P(lane && point) for each lane
+	lanes.forEach((cluster, lane) => {
+            // ignore impossible lanes
+            if (!possibleLanes.has(lane)) {
+                return;
+            }
+            let prob = (1/3) * (1/cluster.points.size);
```

Comment:
I don't quite get the `(1/3)` factor here. This seems like it assumes that there are always three lanes, which is not the case. See Nanisivik CAF RAAS v1, which has "Center" and "East".

Since we normalize a few lines down anyway, I believe this factor has no effect anyway and can be omitted.

Reply:
Good catch, removed this factor.